Task 6/script.js

```diff
@@ -19,3 +19,26 @@ const users = [
   { id: '8', name: 'Simon Peterson', age: 30 },
   { id: '9', name: 'Daniel Cane', age: 51 },
 ];
+
+let sumAge = 0;
```

Comment:
Imperatyvus priėjimas prie problemos. Tai yra akumuliuojantis kintamasis, kuris dabar yra pasiekiamas globaliame scope, nors jis be getUserAverageAge funkcijos prasmės nelabai turi. Vertėtų jį scope'inti pačioje funkcijoje, kad kintamasis nebūtų pasiekiamas išorėje esančios logikos. Rekomenduočiau panaudoti reduce, o ne forEach metodą, taip visa sumavimo logika būtų tilpusi vienoje eilutėje ir teireikėtų rezultatą išsisaugoti kintamajame.